stories/core/Buttons/ButtonGroup.stories.tsx

```diff
@@ -3,25 +3,42 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 import React from 'react';
-import { ButtonGroup, ButtonGroupProps, IconButton } from '../../../src/core';
-import SvgAdd from '@itwin/itwinui-icons-react/cjs/icons/Add';
-import SvgDelete from '@itwin/itwinui-icons-react/cjs/icons/Delete';
-import SvgEdit from '@itwin/itwinui-icons-react/cjs/icons/Edit';
-import SvgUndo from '@itwin/itwinui-icons-react/cjs/icons/Undo';
+import {
+  ButtonGroup,
+  ButtonGroupProps,
+  DropdownMenu,
+  IconButton,
+  MenuItem,
+} from '../../../src/core';
+import {
+  SvgAdd,
+  SvgDelete,
+  SvgEdit,
+  SvgUndo,
+  SvgPlaceholder,
+  SvgMore,
+} from '@itwin/itwinui-icons-react';
 import { Meta, Story } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
+import { CreeveyMeta } from 'creevey';
 
 export default {
   title: 'Buttons/ButtonGroup',
   component: ButtonGroup,
+  argTypes: {
+    style: { control: { disable: true } },
+    className: { control: { disable: true } },
+    id: { control: { disable: true } },
+    children: { control: { disable: true } },
+  },
   parameters: {
-    controls: { hideNoControlsWarning: true },
+    creevey: { skip: ['Overflow'] },
```

Comment:
Didnt want to bother with a test? Would be nice :D

Reply:
Added

Reply:
I meant with an action to show that overflow.. but ok :D